release notes/v0.57.0.md

```diff
@@ -0,0 +1,172 @@
+
+The k6 team has been developing a new official jslib dedicated to functional testing. While it is still under active development and will potentially see breaking changes, the set of APIs and behaviors it offers are meant to make their way into k6 eventually, and it is now available for early feedback.
+
+[k6-testing](https://github.com/grafana/k6-jslib-testing) is a k6 javascript library that offers a seamless way to write functional tests in k6, using a Playwright-compatible assertions API. It exposes an `expect` function, with which assertions can be performed using specific matchers that reflect the expectation.
+When assertions fail, the test will immediately fail with a clear error message, including the expected and actual values in a similar fashion to what users would observe when using Playwright assertions. 
+
+There are many generic matchers (and more to come), such as `toEqual`, `toBe`, or `toBeTruthy`, to only name a few, that can be used to assert conditions during a k6 test.
+
+```javascript
+import { expect } from 'https://jslib.k6.io/k6-testing/0.2.0/index.js';
+
+export default function () {
+    const response = http.get('https://test.k6.io');
+    expect(response.status).toEqual(200);
+    expect(response.body).toBeTruthy();
+    expect(response.json()).toEqual(JSON.stringify({ message: 'Hello, world!' }));
+}
+```
+
+[k6-jslib-testing](https://github.com/grafana/k6-jslib-testing) also includes browser-specific async matchers that wait until the expected condition is met such as `toBeVisible`, `toBeDisabled`, or `toBeChecked`, to name a few.
+
+```javascript
+import { expect } from "https://jslib.k6.io/k6-testing/0.2.0/index.js";
+import { browser } from "k6/browser";
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: "shared-iterations",
+      options: {
+        browser: {
+          type: "chromium",
+        },
+      },
+    },
+  },
+};
+
+export default async function () {
+  const page = await browser.newPage();
+
+  try {
+    // Navigate to the page
+    await page.goto("https://test.k6.io/my_messages.php");
+
+    // Type into the login input field: 'testlogin'
+    const loc = await page.locator('input[name="login"]');
+    await loc.type("testlogin");
+
+    // Assert that the login input field is visible
+    await expect(page.locator('input[name="login"]')).toBeVisible();
+
+    // Expecting this to fail as we have typed 'testlogin' into the input instead of 'foo'
+    await expect(page.locator('input[name="login"]')).toHaveValue("foo");
+  } finally {
+    await page.close();
+  }
+}
+```
+
+It is currently available as part of the [jslib.k6.io](https://jslib.k6.io) repository and is available to use in your k6 tests by adding the following import:
+
+```javascript
+import { expect } from "https://jslib.k6.io/k6-testing/0.2.0/index.js";
+```
+
+Try it out and give us feedback or contribute to the project on the [k6-jslib-testing repository](https://github.com/grafana/k6-jslib-testing)!
```

Comment:
Maybe a question for the team, do we want to include a mention of this new library in the [jslib docs](https://grafana.com/docs/k6/latest/javascript-api/jslib/) page, or do we want to wait for some user feedback/more development before doing so?

Reply:
I've been 😷 and unable to get to it, but I will add documentation for the jslib indeed. This should not be a blocker for the release though 🙇🏻

Reply:
Thanks @oleiade 🙇 . I've created an issue for it here: https://github.com/grafana/k6-docs/issues/1869.